transmission_interface/include/transmission_interface/accessor.hpp

```diff
@@ -46,7 +46,7 @@ std::vector<std::string> get_names(const std::vector<T> & handles)
   std::set<std::string> names;
   std::transform(
     handles.cbegin(), handles.cend(), std::inserter(names, names.end()),
-    [](const auto & handle) { return handle.get_name(); });
+    [](const auto & handle) { return handle.get_prefix_name(); });
```

Comment:
Since we had this issue hidden for some time shows that the code is probably not tested. Can you add a simple test for it?

Reply:
I've added the check in the `AccessorValidation` test from `differential_transmission_test.cpp`. I think it's not the correct place due to it is not related to that type of transmissions. 

What do you think? Where should it be?

Reply:
done!

Reply:
thanks!